src/SpecFlow.NetCore/Fixer.cs

```diff
@@ -4,43 +4,31 @@
 using System.Linq;
 using System.Text;
 using System.Text.RegularExpressions;
+using System.Xml;
 using Specflow.NetCore;
 using static System.Console;
 
 namespace SpecFlow.NetCore
 {
 	internal class Fixer
 	{
-		private readonly string _specFlowExe;
+		private string _specFlowExe;
 		private string _testFramework;
 		private FileInfo[] _featureFiles;
 		private readonly string _toolsVersion;
 
 		public Fixer(string specFlowPath = null, string testFramework = null, string toolsVersion = "14.0")
 		{
-			_specFlowExe = FindSpecFlow(specFlowPath);
-			WriteLine("Found: " + _specFlowExe);
-
+			_specFlowExe = specFlowPath;
 			_testFramework = testFramework;
 			_toolsVersion = toolsVersion;
 		}
 
-		private string FindSpecFlow(string path)
+		private static string FindSpecFlow(string version)

```

Comment:
Reason for `static`?

Reply:
It's like a hint (for those who will read this code) that the method does not use class members.